Tokenizer.cs

```diff
@@ -109,7 +110,15 @@ public class Tokenizer {
 					case ' ':
 					case '\t':
 					case '\r':
-						if(tok.type != TokenType.SKIP_NEWLINE) {
+						if (tok.val == "out" && line.Length > i + 10 && line.Substring(i, 10) == " maneuver:")
+                        {
+							tok.val += line.Substring(i, 10);
+							i += 9;
+
+							HandleToken(new Token(TokenType.OUT_MANEUVER, tok.val, functionName, lineno, col));
+							tok = new Token(functionName, lineno, col + 1);
+						}
+						else if(tok.type != TokenType.SKIP_NEWLINE) {
```

Comment:
This logic shouldn't be handled here. The tokenizer is a stupid machine: it just splits strings. A line such as:

```
out maneuver: circularization "Maneuver to circularize orbit"
```

should be tokenized as:

```
OUT MANEUVER IDENTIFIER STRING
```

Then later in the actual parser we will deal with the `OUT` versus `OUT MANEUVER` case.

But this case is a little tricky. Perhaps it is better to just have `maneuver:` instead of `out maneuver:`.

Reply:
I went with the `maneuver:` syntax
with following tokenization
`out maneuver: circularization "Maneuver to circularize orbit"`
into
`MANEUVER IDENTIFIER STRING`